src/server/bnetserver/Realms/WorldListener.cpp

```diff
@@ -65,7 +66,7 @@ void WorldListener::HandleOpen()
 {
     try
     {
-        _worldSocket->bind(std::string("tcp://*:") + std::to_string(_worldListenPort));
+        _worldSocket->bind(std::string("tcp://"+_bindIp+":") + std::to_string(_worldListenPort));
```

Comment:
This changes the default from * to 0.0.0.0, is there any impact in that or those two are equivalent?